mattermost/mattermost-webapp
# MM-17459 add test for Filtering by first name with Korean characters

Summary
Add test "M17459 Scroll to bottom when sending a message"

Test steps:

1.  Set a user's first name to the following: 정트리나/trina.jung/집단사무국(CO)
2. Log in as another user and view a channel, type in part of  user`s first name
3. verify that suggestion list is visible and has `user-1` value
4. type enter, verify that user mention has been printed
5. type enter, verify that message has been posted with user-1 mention
 
Ticket Link
Jira ticket [MM-17459](https://mattermost.atlassian.net/browse/MM-17459)
closes https://github.com/mattermost/mattermost-server/issues/11755
